client/components/site-thumbnail.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+export default React.createClass({
+  propTypes: {
+    url: React.PropTypes.string
+  },
+  render () {
+    let block = {
+      position: 'absolute',
+      display: 'block',
+      height: '380',
+      width: '100%',
+      background: 'transparent'
+    }
+    return (
+      <div onClick={ this.stopPropergation }>
+        <div style={ block }></div>
+        <iframe src={ this.props.url } width='100%' height='380' border='no' scrolling='no' frameBorder='no'></iframe>
```

Comment:
I'd remove these properties (width/height) and let the code that uses this component set the width and height using CSS.